frontend/components/initiative/Toolbar.vue

```diff
@@ -0,0 +1,76 @@
+<script setup lang="ts">
+import { type InitiativeUserRelationship } from '@/openapi/generated/pacta'
+
+const localePath = useLocalePath()
+const { getMaybeMe } = useSession()
+const { t } = useI18n()
+
+const { isAdmin, maybeMe } = await getMaybeMe()
+const prefix = 'InitiativeToolbar'
+const tt = (key: string) => t(`${prefix}.${key}`)
+
+interface Props {
+  initiativeId: string
+  initiativeUserRelationships: InitiativeUserRelationship[]
+}
+const props = defineProps<Props>()
+
+const isManager = computed(() => {
+  const mm = maybeMe.value
+  return (!!mm && props.initiativeUserRelationships.some(r => r.manager && r.userId === mm.id)) || true
+})
+const isMember = computed(() => {
+  const mm = maybeMe.value
+  return !!mm && props.initiativeUserRelationships.some(r => r.member && r.userId === mm.id)
+})
+const canEdit = computed<boolean>(() => isManager.value || isAdmin.value)
+const canSeeInternal = computed<boolean>(() => isManager.value || isMember.value || isAdmin.value)
+const showToolbar = computed<boolean>(() => canSeeInternal.value || canEdit.value)
+</script>
+
+<template>
+  <div
+    v-show="showToolbar"
+    class="p-buttonset initiative-toolbar"
+  >
+    <LinkButton
+      :to="localePath(`/initiative/${initiativeId}`)"
```

Comment:
Any reason to not have `LinkButton` just always do `localePath`? It'd cut down on a lot of boilerplate, and I think we already detect internal vs external links.

Reply:
^^

Reply:
Ah yes - LinkButton is the only place we could reliably do this, so I think it's better for us not to do so JUST here. It would build up inconsistent expectations across how i18n should be handled on links. It's easier to spot naked links when we know they should always be directly wrapped by an i18n handler.